frontend/app/src/main/java/dk/scheduling/schedulingfrontend/device/Device.kt

```diff
@@ -1,7 +1,78 @@
 package dk.scheduling.schedulingfrontend.device
 
+import java.time.LocalDateTime
+
 data class Device(
     var id: Int,
     var name: String,
     var effect: Int?,
 )
+
+data class DeviceOverview(
+    var device: Device,
+    var event: Event?,
+)
+
+data class Event(
+    var startTime: LocalDateTime,
+    var duration: Long,
+)
+
+fun testDeviceOverview(): List<DeviceOverview> {
```

Comment:
Maybe make testing in another folder/file

Reply:
Fixed